### ISSUE TITLE
[cscli] metrics panic

### ISSUE THREAD

**Issue by LaurenceJJones** [crowdsecurity/crowdsec#2392]
https://github.com/crowdsecurity/crowdsec/blob/9dfc66ef04aaf4a22ffdc88fe21daffeb3900ffc/cmd/crowdsec-cli/metrics.go#L315

We should add a PreRunE to catch if the user is running `cscli metrics` without running the Prometheus locally.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x18d67a4]

goroutine 1 [running]:
main.NewMetricsCmd.func1(0xc001f30300?, {0x3106848?, 0x0?, 0x0?})
    github.com/crowdsecurity/crowdsec/cmd/crowdsec-cli/metrics.go:276 +0x64
github.com/spf13/cobra.(*Command).execute(0xc001f30300, {0x3106848, 0x0, 0x0})
    github.com/spf13/cobra@v1.7.0/command.go:944 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0xc0019c5800)
    github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
    github.com/spf13/cobra@v1.7.0/command.go:992
main.main()
    github.com/crowdsecurity/crowdsec/cmd/crowdsec-cli/main.go:267 +0xe69
```

**Comment by mmetc** [comment]
I can't replicate - we have checks in the runMetrics function, how did you disable the metric exporter?

```
	if err := csConfig.LoadPrometheus(); err != nil {
		return fmt.Errorf("failed to load prometheus config: %w", err)
	}

	if csConfig.Prometheus == nil {
		return fmt.Errorf("prometheus section missing, can't show metrics")
	}

	if !csConfig.Prometheus.Enabled {
		return fmt.Errorf("prometheus is not enabled, can't show metrics")
	}
```

**Comment by LaurenceJJones** [comment]
@mmetc Did you remove the configuration entirely? the user reports when it doesnt exist in config.yaml at all

**Comment by mmetc** [comment]
Ah sorry, I thought you had tested on master. It's been fixed and will be released in the next 1.5.3-rc3
